src/bom/bitcode.rs

```diff
@@ -314,6 +315,12 @@ fn build_bitcode_arguments(chan : &mut mpsc::Sender<Option<Event>>,
 
     let mut bcgen_op = ops.push_op(&run(&*CLANG_LLVM, bc_opts.clang_path)
                                    .set_label("clang:emit-llvm"));
+    match inp_lang {
+        clang_support::Language::Cplusplus => {
+            bcgen_op.set_executable("clang++");
```

Comment:
Does this need any special handling if the clang binary to invoke has a version on it or if there is an environment var override?

Reply:
Oh good catch.  Not an env var override, but we already allow a `--clang` override for plain clang.  Rather than adding yet another command-line argument (or making assumptions about just appending a "++" to the name), I tested and using `-x` seems to work as well, which lets us avoid the above issues.